content/en/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -89,36 +91,48 @@ Only _namespaced_ sysctls can be enabled this way.
 ## Setting Sysctls for a Pod
 
 A number of sysctls are _namespaced_ in today's Linux kernels. This means that
-they can be set independently for each pod on a node. Being namespaced is a
-requirement for sysctls to be accessible in a pod context within Kubernetes.
+they can be set independently for each pod on a node. Only namespaced sysctls
+are accessible in the pod security context within Kubernetes.
 
-The following sysctls are known to be _namespaced_:
+The following sysctls are _namespaced_:
```

Comment:
this list might change from kernel to kernel. This was the reason for the "known to be".

Reply:
Thanks for the clarification. I misunderstood the intent.